.trivyignore

```diff
@@ -5,6 +5,8 @@ CVE-2023-43495
 CVE-2023-43496
 CVE-2023-36478
 CVE-2023-5072
+# Temporary add ignore to build rock
+CVE-2024-22233
```

Comment:
Can this be removed?

Reply:
I removed the comment, The CVE ignore still needs to be there for the rock to be built I think